third-party/test/index.ts

```diff
@@ -52,10 +56,23 @@ if (!tty.getWindowSize) {
     };
 }
 
-let mocha = new Mocha({
-    ui: "bdd",
-    useColors: true,
-});
+// 2020-02-05: Amazon addition.
+export function defaultMochaOptions(useColors: boolean = true): MochaOptions {
+    return {
+        ui: "bdd",
+        useColors: useColors,
```

Comment:
Slight style nit: `useColors: useColors,` can be shortened to `useColors,` Up to you if you want to change it.